content/docs/user-guide/action-manager/fundamentals/architecture/visibility.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Action Visibility
+linktitle: Visibility
+description: 
+weight: 204
+---
+
+The visibility properties of an action determines when actions are displayed in the User Interface as part of a toolbar or a menu based on their active state and assigned Action Context Modes.
+These behaviors are dynamic, as in they will be re-evaluated whenever the state of the Editor changes and an action's enabled state is toggled, or the currently active Action Context Mode changes.
+
+
+## Visibility Setting Values
+
+The Visibility properties of an action can be set to one of three values.
+
+
+#### AlwaysShow
+
+The Action will always be shown as part of the menus or toolbars it is assigned to.
+In case the action's enabled state is set to false, or the action is assigned to action context modes different from the currently active one, the action will be visible but greyed out/inaccessible.
+
+
+#### OnlyInActiveMode
+
+The Action will only be shown if it was assigned to the currently active Action Context Mode, or to no Action Context Mode.
+In case the action's enabled state is set to false, the action will be visible but greyed out/inaccessible.
+If the action is assigned to Action Context Modes different from the currently active one, the action will not appear in the menu or toolbar.
+
+
+#### HideWhenDisabled
+
+The Action will only be shown if it was assigned to the currently active Action Context Mode, or to no Action Context Mode, and its enabled state is set to true.
+In case the action's enabled state is set to false, or the action is assigned to Action Context Modes different from the currently active one, the action will not appear in the menu or toolbar.
+The action will never be displayed as greyed out/inaccessible.
```

Comment:
```suggestion
#### `HideWhenDisabled`

The action only shows if it was assigned to the currently active action context mode, or to no action context mode, and its enabled state is set to true.
In case the action's enabled state is set to false, or the action is assigned to action context modes that are different from the currently active one, the action will not appear in the menu or toolbar.
The action is never be displayed as greyed out and inaccessible.
```